content/chainguard/migration/migration-guides/migrating-python.md

```diff
@@ -28,17 +28,17 @@ Chainguard Images are built on [Wolfi](/open-source/wolfi/), a [distroless](/sof
 {{< blurb/wolfi >}}
 {{< /details >}}
 
-Because Chainguard Images aim to be minimal, including providing separate development and production tags, adapting your containerized application requires that you consider some additional factors that will be discussed below.
+Because Chainguard Images aim to be minimal, adapting your containerized application requires that you consider some additional factors that will be discussed below.
```

Comment:
```suggestion
Because Chainguard Containers aim to be minimal, adapting your containerized application requires that you consider some additional factors that will be discussed below.
```